medical_prescription_state/models/medical_prescription_order.py

```diff
@@ -7,5 +7,8 @@
 
 class MedicalPrescriptionOrder(models.Model):
     """ Add Kanban functionality to MedicalPrescriptionOrder """
-    _inherit = ['medical.prescription.order', 'medical.base.kanban']
+    _inherit = ['medical.prescription.order', 'base.kanban.abstract']
     _name = 'medical.prescription.order'
+
+    def _default_stage_id(self):
```

Comment:
Centralize this by putting in core. Instead of `return`, you should `return self`